pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -769,13 +769,21 @@ func (m *migrationMonitor) startMonitor() {
 
 // logMigrationInfo logs the same migration info as `virsh -r domjobinfo`
 func logMigrationInfo(logger *log.FilteredLogger, uid string, info *libvirt.DomainJobInfo) {
+	bToMB := func(bytes uint64) uint64 {
+		return bytes / 1000000
+	}
+
+	bToMbps := func(bytes uint64) uint64 {
+		return bytes / 125000
```

Comment:
can you please explain this number?

Reply:
That's the constant to convert bytes to Megabits.

Reply:
Ah, clearer now :) thanks!